src/Microsoft.SqlTools.ServiceLayer/Agent/AgentService.cs

```diff
@@ -298,7 +298,8 @@ internal async Task HandleJobActionRequest(AgentJobActionParams parameters, Requ
                 catch (Exception e)
                 {
                     result.Success = false;
-                    result.ErrorMessage = e.Message;
+                    result.ErrorMessage = e.Message + Environment.NewLine + "\t" + e.InnerException.Message
```

Comment:
shouldn't we check that `InnerException` isn't null prior to accessing it?  Or for some reason we know it will always be non-null here?

Reply:
Usually there's always an inner exception (which is the indented exception details shown in SSMS). But I'll add the check to be safe.

Reply:
When accessing a member without a null check we need to be 100% sure it's not null, so if it's "usually" not null we still need a null check.  FYI, in general most exceptions have InnerException null, though SqlExeptions often do have an InnerException.

Unless you have a block of code that sets the innerexception that you know runs prior to your catch block, you should always null check prior to access.